### ISSUE TITLE
Failed to resolve circular dependency

### ISSUE THREAD

**Issue by zecar** [nuxt-modules/i18n#2078]
### Environment

------------------------------
- Operating System: Darwin
- Node Version:     v14.19.1
- Nuxt Version:     3.4.3
- Nitro Version:    2.4.0
- Package Manager:  npm@9.6.6
- Builder:          vite
- User Config:      app, modules, css, postcss
- Runtime Modules:  @pinia/nuxt@0.4.10, @nuxtjs/i18n@8.0.0-beta.12
- Build Modules:    -
------------------------------

### Reproduction

repo -> https://github.com/zecar/i18n-issue
steps:
install deps
open two tabs both with http://localhost:3000/
run yarn dev
you should see the error

if you close the server, leave only one browser tab opened and run yarn dev it should work fine

### Describe the bug

i get this error
![image](https://github.com/nuxt-modules/i18n/assets/8117818/b71d4cbc-861d-475b-acac-4a006641a6ab)

### Additional context

i did the initial setup of the project with multiple restarts of the server and it worked great.. i did a git commit and after i started once more the server and I get this error

### Logs

```shell
[vite-node] Failed to resolve circular dependency, stack: - /Users/ionutcapraru/work/github/nuxt-frontend/.nuxt/i18n.options.mjs - virtual:nuxt:/Users/ionutcapraru/work/github/nuxt-frontend/.nuxt/i18n.internal.mjs - @nuxtjs/i18n/__config__?target=../i18n.config.ts&hash=2fc1c60d&from=/Users/ionutcapraru/work/github/nuxt-frontend/.nuxt/i18n.options.mjs - /Users/ionutcapraru/work/github/nuxt-frontend/.nuxt/i18n.options.mjs - /Users/ionutcapraru/work/github/nuxt-frontend/node_modules/@nuxtjs/i18n/dist/runtime/plugins/i18n.mjs - virtual:nuxt:/Users/ionutcapraru/work/github/nuxt-frontend/.nuxt/plugins/server.mjs - /Users/ionutcapraru/work/github/nuxt-frontend/node_modules/nuxt/dist/app/entry.js
```

**Comment by zecar** [comment]
fixed by: not having 2 tabs of the project opened at the same time.....

**Comment by kazupon** [comment]
Could you give us your minimul reproduction repo or stackblize, please? 🙏

**Comment by zecar** [comment]
> Could you give us your minimul reproduction repo or stackblize, please? 🙏

I updated the reproduction steps to include a repo and reproduction steps

**Comment by zecar** [comment]
> Could you give us your minimul reproduction repo or stackblize, please? 🙏

Can you remove the "need reproduction" badge? I have provided reproduction steps

**Comment by ewilan-riviere** [comment]
I update my `nuxt` version to `3.5.0` and it's work for me.

**Comment by dargmuesli** [comment]
@zecar can you confirm that updating to nuxt v3.5.0 solves the issue?

**Comment by xjimdim** [comment]
can someone confirm that this got fixed in 3.5.0?

**Comment by kazupon** [comment]
close, in-activity and v8.0.0-beta.13 was released, so that release has changed locale handling and build process. I seem that this issue is fixed.
If you have still issue, please you will open as new issue.

thanks!
